main.tf

```diff
@@ -106,3 +106,13 @@ resource "azurerm_mysql_flexible_server_firewall_rule" "mysql_flexible_server_fi
   start_ip_address    = local.mysql_flexible_server_firewall_rule[each.key].start_ip_address
   end_ip_address      = local.mysql_flexible_server_firewall_rule[each.key].end_ip_address
 }
+
+resource "azurerm_mysql_flexible_server_active_directory_administrator" "mysql_flexible_server_active_directory_administrator" {
+  for_each = var.mysql_flexible_server_active_directory_administrator
+
+  server_id   = var.mysql_flexible_server_active_directory_administrator[each.key].server_id
```

Comment:
for the other resources, there were mainly locals used. I think for consistency, this should be adapted

Reply:
The README.md will be automatically updated via the GitHub action teraform-docs once the PR has been merged. For a useful example of mysql_flexible_server_active_directory_administrator in full_main.tf, additional resources (e.g. user_managed_identity) are required, which would have to be provided using other modules. I'll see if I can add an example.

For the other resources in this module, default values for optional arguments are set in locals. For this resource, all arguments are necessary for mysql_flexible_server_active_directory_administrator (see https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/mysql_flexible_server_aad_administrator#arguments-reference). That's why I didn't use locals here.

Reply:
Thanks for the thorough explanation! :slightly_smiling_face: